modules/starting-a-basic-workbench.adoc

```diff
@@ -0,0 +1,82 @@
+:_module-type: PROCEDURE
+
+[id='starting-a-basic-workbench_{context}']
+= Starting a basic workbench
+
+[role='_abstract']
+Using the basic workbench is based on a server-client architecture. The basic workbench runs in a container on the Red Hat OpenShift cluster. The client is the IDE interface that opens in your web browser on your local computer. However, all of the commands that you enter in the IDE are executed by the workbench. This architecture allows you to interact through your local computer in a browser environment, while all processing occurs on the cluster. The cluster provides the benefits of larger available resources and security because the data being processed never leaves the cluster.
+
+From the *Start basic workbench* tile, you can start a basic workbench. If you require extra power for use with large datasets, you can assign accelerators to your workbench to optimize performance. 
+
+.Prerequisites
+* You are logged in to {productname-long}.
+* You are starting the workbench for the first time or you stopped your workbench.
+* You know the names and values you want to use for any environment variables in your workbench environment, for example, `AWS_SECRET_ACCESS_KEY`.
+* If you want to work with a large data set, work with your administrator to proactively increase the storage capacity of your workbench. If applicable, also consider assigning accelerators to your workbench. 
+
+.Procedure
+
+. In the left navigation pane, click *Applications* → *Enabled*.
+. On the *Enabled* page, locate the *Start basic workbench* tile.
+
+. Click *Launch application*.
++
+If you see an *Access permission needed* message, you are not in the default user group or the default administrator group for {productname-short}.
+ifndef::upstream[]
+Ask your administrator to add you to the correct group by using link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing-users-and-groups#adding-users-to-user-groups_managing-rhoai[Adding users to {productname-short} user groups].
+endif::[]
+ifdef::upstream[]
+Ask your administrator to add you to the correct group.
+endif::[]
++
+If your credentials are accepted, the *Workbench control panel* opens displaying the *Start a basic workbench* page.
+. Start a basic workbench.
+
+.. In the *Workbench image* section, select the workbench image to use for your server. 
++
+Different workbench images have different packages installed by default. Click the help icon (?) next to a workbench image name to view a list of its included packages.
+
+.. If the workbench image contains multiple versions, select the version of the workbench image from the *Versions* section.
++
+[NOTE]
+--
+When a new version of a workbench image is released, the previous version remains available and supported on the cluster. This gives you time to migrate your work to the latest version of the workbench image.
+--
+.. From the *Container size* list, select a suitable container size for your server.
```

Comment:
server? thought we stopped using the workbench server term

Reply:
seconded, 

`select a suitable container size for your workbench.`